### PR TITLE
[RHACS][Docs][4.6 and later] ROX-27026: Improvements in Violations documentation

### PR DESCRIPTION
**Pull request by agantony**
Version(s):

Issue:
https://issues.redhat.com/browse/ROX-27026

Link to docs preview:
https://85046--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/respond-to-violations.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.6`

### CODE REVIEW

File: modules/analyzing-all-violations.adoc [hunk 1]
```diff
@@ -11,6 +11,10 @@ By viewing the *Violations* page, you can analyze all violations and take correc
 .Procedure
 
 . In the {product-title-short} portal, click *Violations*.
+. To navigate through the different types of violations, do any of the following tasks:
```
Comment (snarayan-redhat) [inline modules/analyzing-all-violations.adoc:14]:
```suggestion
. To navigate through the different types of violations, perform any of the following tasks:
```

**Review by snarayan-redhat — COMMENTED** [review]
Just one nit! Otherwise, looks good.

File: modules/analyzing-all-violations.adoc (hunk 1 above)
Comment (agantony) [reply modules/analyzing-all-violations.adoc:14]: Thanks for the suggestion, however, I would prefer to keep it unchanged to maintain consistency across ACS docs. I don't think there is any style guide restriction on the usage of "do".

**Review by sachaudh — APPROVED** [review]

**Review by charmik-redhat — APPROVED** [review]

**Pull request merged**